src/ProfileEndpoints.jl

```diff
@@ -315,6 +316,28 @@ function handle_task_backtraces(stage_path = nothing)
     return HTTP.Response(200, backtrace_file)
 end
 
+###
+### Profile removal
+###
+
+function has_profile_extension(file)
+    return endswith(file, ".profile") || endswith(file, ".pb.gz") || endswith(file, ".task_backtraces")
```

Comment:
Does this cover everything that gets dumped? Maybe include the heapsnapshot as well?

Reply:
Dropped this function in latest commit.